modules/ipp/scanner.go

```diff
@@ -341,6 +396,15 @@ func (scanner *Scanner) augmentWithCUPSData(scan *scan, target *zgrab2.ScanTarge
 		return zgrab2.NewScanError(zgrab2.SCAN_APPLICATION_ERROR, ErrVersionNotSupported)
 	}
 
+	if cupsResp.StatusCode != 200 {
+		return zgrab2.NewScanError(zgrab2.SCAN_APPLICATION_ERROR, errors.New("Response returned with status " + cupsResp.Status))
```

Comment:
Are all responses > 200 && < 400 really invalid? Or even 4xx, is it possible that you get say a 403, but it returns some attributes in the body nonetheless?